ntbs-service/DataMigration/NotificationImportHelper.cs

```diff
@@ -0,0 +1,44 @@
+using System;
+using Microsoft.Extensions.Configuration;
+using System.Data.SqlClient;
+using Dapper;
+using System.Threading.Tasks;
+
+namespace ntbs_service.DataMigration
+{
+    public interface INotificationImportHelper
+    {
+        Task CreateTableIfNotExists();
+        string GetImportedNotificationsTableName();
+    }
+
+    public class NotificationImportHelper : INotificationImportHelper
+    {
+        private readonly string connectionString;
+        private readonly string importedNotificationsTableName;
+        private readonly string createImportedNotificationsTableQuery = @"
+            IF (NOT EXISTS (SELECT * FROM INFORMATION_SCHEMA.TABLES WHERE TABLE_NAME = '{0}'))
+            CREATE TABLE {0} (
+                LegacyId varchar(255) NOT NULL PRIMARY KEY,
+                ImportedAt datetime NOT NULL
+            )";

```

Comment:
We should leave a comment explaining why we're using such unconventional measures!